pkgs/by-name/in/inotify-info/package.nix

```diff
@@ -1,28 +1,31 @@
-{ lib, stdenv, fetchFromGitHub }:
+{
+  lib,
+  stdenv,
+  fetchurl,
+}: let
+  version = "0.0.1";
+in
+  stdenv.mkDerivation (finalAttrs: {
+    name = "inotify-info";
+    inherit (version);
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "inotify-info";
-  version = "unstable-2024-01-05";
+    src = fetchurl {
```

Comment:
can we go back to fetchFromGithub ?

Reply:
Github [does not promise to keep the generated hashes stable](https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/). That is the reason why I generated the release archives in the upstream project.

`fetchFromGitHub` relies on those hashes unchanged.

Reply:
@motiejus the article you have linked is not related to git commit revisions